### ISSUE TITLE
Should we develop and implement the ability to search by district, sub district and civil parish or all fields from the census information

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1034]
Cen1 permits searching by all fields of the census information. 
The initial implementation of Cen2 permits only searching by the sub district place. The CSV implementation of Cen2 permits searching by the civil parish place. Hence CEN2 as presently constructed  will have a mixture of civil parish and subdistrict places.
**We could have the CSV entries only searched by sub district place if consistency is preferred.**
On the longer term should we develop searching by district, sub district and civil parish places.
Or should we revert to searching by any or all of the fields of the census information. 

E3 U3 S6 C7: P19

**Comment by PatReynolds** [comment]
Checking my understanding, @Captainkirkdawson  so option 1 is to serve the information of the subdistricts available as FC1 transcriptions, and the civil parishes available as FC2 transcriptions?  all in one list, or as two lists on the search form? Will selections  of (e.g.) one subdistrict and two civil parishes be permitted?

**Comment by Captainkirkdawson** [comment]
@PatReynolds your understanding is correct; cen1 style transcription are by subdistrict places while cen2 style csv transcription as presently implemented are by civil parish places.  One list. One selection

**Comment by PatReynolds** [comment]
@PatReynolds to think about this - who should be asked / UX 

@DeniseColbert to look at feedback on this issue

**Comment by geoffj-FUG** [comment]
I believe that we have now been able to resolve the conflicts identified above.
The closure of vld uploads has enabled us to reconcile the civil parish information recorded in vlds with the PARMS.
This has involved adjusting over 20,000 entries (a under-estimate I am sure) to align the vld data collection with the rules applied to the csv data collection, as far as civil parishes are concerned.
Once the reconciliation is complete we will have a civil parish data set aligned with the PARMS. This means that we will be able to proceed with the implementation of searching at a level below the sub Registration District.
This level of searching is effectively the level of searching the FC1 reached. It is the lowest level of place information that was collected. The lack of testing against the PARMS means that sometimes the civil parish was recorded, sometimes the hamlet or township was recorded and sometimes entries were made that had absolutely no relationship to the enumerated information were recorded. The size of the data available hid any discrepancies.
Do you want me to proceed with a definition for the extension of searching from the sub registration district level to the civil parish level?
Geoff

**Comment by geoffj-FUG** [comment]
The civil parish reconciliation of the vld data collection is almost complete. 
The current search jumps from County level to civil parish level. We cannot go below this level.
The Registration District level sits between these two levels:
County ... Registration District ... Piece (Sub District) ... Civil Parish.
Registration Districts have several civil parishes that can be extracted from the PARMS.

We are now looking at having one universal search that will cover all three systems. There is a distinct advantage in developing the Registration District / Hundred search level because it aligns with the FreeBMD Data Set. That means that both systems could search on the same Registration District entry.
The civil parish search could then be refined by applying the Registration District filter so that only the civil parishes in that Registration District are displayed instead of all te civil parishes within the chosen county or counties.

Should we proceed to define the system based on this premise?

Geoff.

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to talk to Martin about Lat/Long (FreeBMD) and the wider discussion of this needs to be taken to the (yet to be formed) steering group.

**Comment by PatReynoldsFUG** [comment]
For clarification: 
Two stories here that need separate discussion
a) the ability to search from a single interface across all three databases (and potential future databases) using the latitde and longditude of a place (a place can be a hamlet or a country or anything in between#0.
b) the ability to search FreeCEN with any of the place name levels used in the various Censuses, as independently as possible - e.g. civil parish within a town without town or county.


(b) is one of a number of stories that will be needed to enable (a)

As the title has not really been discussed here, should I make that a new story, and retitle this one "Enable Cross-database searching"?

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to make/find Systemwide story for (a) and then this can be closed

**Comment by PatReynoldsFUG** [comment]
(a) is systemwide story #292

Moving this to ice for (b) awaiting decision on Systemwide#292

**Comment by geoffj-FUG** [comment]
This is included in the specification for the new search form.

Geoff
